### ISSUE TITLE
Guidelines 364: If the software supports the export of the statistical analysis steps (for example R scripts) these are submitted as supplementary data

### ISSUE THREAD

**Issue by mlocardpaulet** [jgriss/ReproducibleMSGuidelines#31]
or made publicly available? For example through gitHub?

**Comment by veitveit** [comment]
and should have a version when on GitHub. We need to align all guidelines that are related to GitHub or alike: 
- has to be there
- needs to have a assigned version number
- needs to be on an institutional account

**Comment by jgriss** [comment]
I agree with @veitveit we absolutely must double check all items and make sure we are consistent thoughout
